enterprise/utils.py

```diff
@@ -438,7 +409,40 @@ def get_notification_subject_line(course_name, template_configuration=None):
         return stock_subject_template.format(course_name=course_name)
 
 
-def send_email_notification_message(user, enrolled_in, enterprise_customer, email_connection=None):
+def find_enroll_email_template(enterprise_customer, template_type):
+    """
+    Find email template from the template database represented by EnrollmentNotificationEmailTemplate model.
+
+    Returns:
+      Customer specific template if found.
+      Default template for the given type if found.
+      None if neither default template, nor per customer template found.
+    """
+    enrollment_template = enroll_notification_email_template()
+    # first try customer specific template for this type
+    try:
+        enterprise_template_config = enterprise_customer.enterprise_enrollment_template
+    except (ObjectDoesNotExist, AttributeError):
+        enterprise_template_config = None
+
+    if not enterprise_template_config:
+        # use the fallback template instead
+        template_queryset = enrollment_template.objects.filter(
+            enterprise_customer=None,
+            template_type=template_type,
+        )
+        enterprise_template_config = template_queryset.first()
```

Comment:
Can't you just attach `[0]` or `first()` on the previous equation instead of having to make 2 separate variables?